autonomy/data/Dockerfiles/agent/Dockerfile

```diff
@@ -1,34 +1,19 @@
-ARG AEA_VERSION=latest
-FROM valory/open-aea-user:${AEA_VERSION}
-ARG AUTHOR=default_author
-ARG AEA_AGENT
+ARG AUTONOMY_IMAGE_VERSION="latest"
+ARG AUTONOMY_IMAGE_NAME="valory/open-autonomy"
 
-COPY openssl.cnf /etc/ssl
-COPY Pipfile .
-COPY . /home/ubuntu
+FROM ${AUTONOMY_IMAGE_NAME}:${AUTONOMY_IMAGE_VERSION}
```

Comment:
The args actually need to be under the from command, otherwise you lose them in the next stage

Reply:
We need `AUTONOMY_IMAGE_VERSION` and `AUTONOMY_IMAGE_NAME` to pull the base image as you can see 

```Dockerfile
FROM ${AUTONOMY_IMAGE_NAME}:${AUTONOMY_IMAGE_VERSION}
```

We don't use them elsewhere so it's okay for them to be at the top